configs/wavemaker.json

```diff
@@ -1,10 +1,10 @@
 {
   "index_name": "wavemaker",
   "start_urls": [
-    "https://www.wavemaker.com/learn/"
+    "https://docs.wavemaker.com/learn/"
   ],
-  "sitemap_urls": [
-    "https://www.wavemaker.com/learn/sitemap.xml"
+  "stop_urls": [
+    "https://www.wavemaker.com/learn/"
```

Comment:
```suggestion
```

The scraper will only follow the domain given in the `start_urls`

Reply:
removed the stop_urls key from the json